app/controllers/organisations_controller.rb

```diff
@@ -29,7 +47,15 @@ def user_belongs_to_same_org
     current_organisation == @organisation
   end
 
+  def assign_user_to_organisation(organisation)
+    current_user.organisations << organisation
```

Comment:
We only want to do this if the organisation has no users associated with it right?
Should we guard for that here?

Reply:
NM

Reply:
Yeah it can only be a brand new organisation at this point